src/nvim/search.c

```diff
@@ -2121,7 +2121,7 @@ static int check_linecomment(char_u *line)
  * Show the match only if it is visible on the screen.
  * If there isn't a match, then beep.
  */
-void 
+void
```

Comment:
these whitespace changes are triggering the linter :)

Reply:
Should I put them back?